medical_prescription/tests/test_medical_prescription_order_line.py

```diff
@@ -43,3 +44,18 @@ def test_name_search_none(self):
             len(recs),
             'Length of recs (%d) not equal to 3' % (len(recs)),
         )
+
+    def test_prescription_order_id_required(self):
```

Comment:
We don't need to test ORM stuff like this - it's tested elsewhere in model tests (we get a free 50-60% test coverage in stuff that's covered abstract). Basically don't worry about testing anything you didn't create a method for.